tests/e2e/pages/vscode-desktop.page.ts

```diff
@@ -278,4 +279,26 @@ export class VSCodeDesktop extends VSCode {
     }
     return this.window;
   }
+
+  public async SetUpSolutionServer(toEnableSolutionServer = false) {
+    const url = process.env.SOLUTION_SERVER_URL;
+    if (!this.repoDir) {
+      throw new Error('Missing repository path.');
+    }
+    if (!url) {
+      throw new Error('Missing Solution Server URL');
+    }
+    const config = getSOlutionServerConfig(toEnableSolutionServer);
+    writeOrUpdateSettingsJson(path.join(this.repoDir ?? '', '.vscode', 'settings.json'), config);
```

Comment:
Can you replace that method for `openWorkspaceSettingsAndWrite` ? It does the same thing but through the UI so later it will work in VSCode as well.

And then you can move this method to the main VSCode class